templates/kube-scheduler.yaml

```diff
@@ -48,6 +48,10 @@ spec:
       value: {{ext_scheduler_label_keys}}
     - name: LOG_LEVEL
       value: {{ext_scheduler_log_level}}
+    - name: DECC_SYSTEM_NAMESPACES
```

Comment:
The master role is independent of DECC, so we shouldn't name the vars as `DECC*`. So may be SYSTEM_NAMESPACES and SYSTEM_POD_LABEL would be appropriate?